### DISCUSSION TITLE
Lesson-7 | Part-6

### DISCUSSION THREAD

**Discussion by vigneshbalu96** [Cyfrin/foundry-full-course-cu#1986] [General]
Hello Blockchaineers, :smile: Hope all are doing good!

While running the following command to run the deploy script:

```sh
forge script script/DeploySimpleStorage.s.sol --rpc-url http://127.0.0.1:8545 --broadcast --private-key 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
```
I encountered the following error:
```
~/foundry-f24/Foundry_Fund_Me_F24$ forge script script/DeployFundMe.s.sol --rpc-url http://127.0.0.1:8545 --broadcast --private-key 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
[⠒] Compiling...
No files changed, compilation skipped
2024-07-11T03:45:59.473399Z ERROR foundry_compilers_artifacts_solc::sources: error="/home/vignesh/foundry-f24/Foundry_Fund_Me_F24/src/Counter.sol": No such file or directory (os error 2)
Error: 
failed to read artifact source file for `/home/vignesh/foundry-f24/Foundry_Fund_Me_F24/src/Counter.sol`

Context:
- "/home/vignesh/foundry-f24/Foundry_Fund_Me_F24/src/Counter.sol": No such file or directory (os error 2)
```

I literally don't have any `Counter.sol` file inside my project folder, and the `foundry.toml` file is also not configured to point to `Counter.sol`. Why is it pointing to this file? 🤔

Thanks for taking your time to reply...

**Comment by EngrPips** [comment]
Hello @vigneshbalu96, run `forge clean`, then run `forge build`, and then retry running `forge script script/DeploySimpleStorage.s.sol --rpc-url http://127.0.0.1:8545 --broadcast --private-key 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80`

Let's know how it goes.

**Reply by nox-dave** [reply]
Had the same problem and this solved it for me. Thank you!

**Reply by EngrPips** [reply]
Awesome. Keep crushing it.

**Reply by MohammedHam24** [reply]
Awsome, Thank you

**Reply by EngrPips** [reply]
Anytime, Friend.
